readme.md

```diff
@@ -155,8 +155,31 @@ The code in this library has only been tested on Lucee and likely doesn't work o
 
 ## Extending this module/service
 
-Individuals who would like to contribute to this library can follow the steps below to configure this tool to run independently for the purposes of extending and testing the code:
-
-1. This package currently depends on a `cfconfig.local` host entry.  You can add this to your system's `hosts` file manually or leverage a tool like [Commandbox HostUpdater](https://www.forgebox.io/view/commandbox-hostupdater) to add the host entry.
-2. Start up this project with CommandBox using `server start` - this will open a browser to the test runner for this project.
-3. Happy Contributing!
+Individuals who would like to contribute to this library can follow the steps below to configure this tool to run independently for the purposes of extending and testing the code.
+
+Happy contributing!
+
+### Prerequisites
+
+ * This package depends on a `cfconfig.local` host entry.  You can add this to your system's `hosts` file manually or leverage a tool like [Commandbox HostUpdater](https://www.forgebox.io/view/commandbox-hostupdater) to add the host entry.
+   * Manual
+     * Hosts File Location
+       * Windows: `C:\Windows\System32\drivers\etc\hosts`
+       * Mac/Linux: `/etc/hosts`
+     * Add the following line to the file:
+       ```
+       127.0.0.1 cfconfig.local
+       ```
+ * Install [CommandBox](https://www.ortussolutions.com/products/commandbox)
+ * Install CommandBox Packages
+   ```
+   box install adobe-password-util
+   box install coldbox
+   box install lucee-password-util
+   box install PropertyFile
+   box install testbox
```

Comment:
Why do I get this kind of error if I do `box install packageA packageB packageC`?

```
ERROR (6.1.0+00813)

Parameter [save] has a value of ["testbox"] which is not of type [boolean].
```

Reply:
Because that's not the right syntax.  The install command accepts a comma-delimited list of packages
```
box install packageA,packageB,packageC
```

Reply:
Thanks. Side note: I think `box install help` doesn't specify how to install multiple packages.